token/services/ttx/transaction.go

```diff
@@ -9,6 +9,8 @@ package ttx
 import (
 	"encoding/asn1"
 
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/vault/keys"
+
```

Comment:
please remove empty line and run go imports

Reply:
done